app/models/manageiq/providers/ansible_tower/automation_manager/amazon_credential.rb

```diff
@@ -1,2 +1,5 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::AmazonCredential < ManageIQ::Providers::AnsibleTower::AutomationManager::CloudCredential
+  ATTRIBUTES = [:description, :username, :security_token].freeze
```

Comment:
Not crazy about calling it `ATTRIBUTES`, I don't want it to be confused with `#attributes`.  Maybe `EXTRA_ATTRIBUTES`?